gulpfile.js

```diff
@@ -94,29 +95,40 @@ gulp.task('imgmin', function () {
       .pipe(gulp.dest('public/img'));
 });
 
-// webpack build:
-gulp.task('webpack', function() {
-  webpack({
-    entry: './js/main.js',
-    output: {
-      path: __dirname + '/public/js',
-      filename: 'j.js'
-    },
-    plugins: [
-      new webpack.optimize.UglifyJsPlugin({
+// wrapper for JS build, supporting both
+// minified and unminified versions
+function jsBuild(isProduction) {
+  return gulp.src('js/main.js')
+    .pipe(webpackStream({
+      entry : {
+        j : './js/main.js'
+      },
+      output: {
+        filename: '[name].js'
```

Comment:
I'm wondering what this `[name].js` is?

Also, thank you so much for doing this :) I love that it only minifies in production